src/components/UserProfile.js

```diff
@@ -87,28 +88,18 @@ function UserProfile() {
   return (
     <div>
       {error ? <ErrorModal errorMsg={error} setError={setError} /> : null}
+      {deleteAccountAttempt ?
+        <ProfileDeleteModal
```

Comment:
my modal is finally good enough for keon's profile 😭

Reply:
It was always good enough!